### PR TITLE
Update trinket_m0_haxpress.md

### PR DESCRIPTION
**Pull request by Timeline8**
Links are now all dead links.

### CODE REVIEW

**Review by FoamyGuy — CHANGES_REQUESTED** [review]
There is an archived version available here: https://web.archive.org/web/20210727135908/http://daveastels.com/trinket-m0-express-hack.html It's missing some of the original images, but it's got a lot of information for anyone who would want to create one of these. 

I think it'd be good to link to that one and note it in the text as an archived version.

**Comment by Timeline8** [comment]
Updated changes with new archived URL. Not sure if I did it right for the Board URL line though.

**Review by Timeline8 — COMMENTED** [review]
Updated links to use the archived version of the old web page no longer online.

File: _board/trinket_m0_haxpress.md [hunk 1]
```diff
@@ -4,8 +4,7 @@ board_id: "trinket_m0_haxpress"
 title: "Trinket M0 Haxpress Download"
 name: "Trinket M0 Haxpress"
 manufacturer: "Dave Astels"
-board_url:
- - "http://daveastels.com/trinket-m0-express-hack.html"
+board_url: https://web.archive.org/web/20210727135908/http://daveastels.com/trinket-m0-express-hack.html
```
Comment (FoamyGuy) [inline _board/trinket_m0_haxpress.md:7]: I think it's best to make formatting match all of the other device config files with the board URL on it's own line with a hyphen at the beginning.
```
 - "https://web.archive.org/web/20210727135908/http://daveastels.com/trinket-m0-express-hack.html"
```

I'm not entirely certain if that syntax is 100% necessary, and I also don't know for sure where this board_url value gets used, but I think keeping it consistent across devices whenever possible is best.

**Review by Timeline8 — COMMENTED** [review]
Agreed. Looks good to me.

**Review by Timeline8 — COMMENTED** [review]
OK I think it looks good now.

**Review by FoamyGuy — APPROVED** [review]
Thank you!

**Pull request merged**